doc/manuals/orion-api.md

```diff
@@ -68,11 +68,13 @@
     - [Notification Messages](#notification-messages)
     - [Custom Notifications](#custom-notifications)
       - [Macro substitution](#macro-substitution)
+      - [Headers special treatment](#headers-special-treatment)
+      - [Remove headers](#remove-headers)
+      - [Text based payload](#text-based-payload)
       - [JSON payloads](#json-payloads)
+      - [NGSI payload patching](#ngsi-payload-patching)
       - [Omitting payload](#omitting-payload)
-      - [Remove headers](#remove-headers)
       - [Additional considerations](#additional-considerations)
-      - [Custom payload and headers special treatment](#custom-payload-and-headers-special-treatment)
```

Comment:
As far as I checked, all the references to #custom-payload-and-headers-special-treatment has been fixed in this PR also, but it would be great a double check @mapedraza

Reply:
It is much more clear now. Should we include a group for "payload transformation" and another for "headers transformation", leaving the index like this?


- [Custom Notifications](#custom-notifications)
   - [Macro substitution](#macro-substitution)
   - [Payload transformation]()
     - [Text based payload](#text-based-payload)
     - [JSON payloads](#json-payloads)
     - [NGSI payload patching](#ngsi-payload-patching)
     - [Omitting payload](#omitting-payload)
   - [Headers transformation]()
     - [Headers special treatment](#headers-special-treatment)
     - [Remove headers](#remove-headers)
   - [Additional considerations](#additional-considerations)

Reply:
I also though on that possibility... but at the end I think adding an extra sub-division level can be more complex for the user and by the moment we don't have so many level-2 sub-sections to need an extra level here.